README.md

```diff
@@ -114,7 +114,7 @@ Make sure changes made from this commmand are merged back into master before rel
 yarn release
 ```
 
-5. Push the branch you created, and make a pull request. It will contain all `package.json` changes for your publish.
+5. Push the tags from the release up to Github.
```

Comment:
Isn't this step done automatically?

Reply:
Don't believe so, I got this from the changesets documentation. But I could be wrong.

Reply:
We should specify that you can push the tags straight to master then, right?